yt/utilities/lib/volume_container.pxd

```diff
@@ -27,9 +39,25 @@ cdef struct VolumeContainer:
     np.float64_t idds[3]
     int dims[3]
 
+#-----------------------------------------------------------------------------
+# vc_index(VolumeContainer *vc, int i, int j, int k)
```

Comment:
My assumption is this is a way to denote a specific voxel in the volume container. Not sure how the match checks out though. Can you explain that?

Also, what is ```spos``` in vc_pos_index

Reply:
So I think we don't use `vc_pos_index` in the volume rendering.  The reason for this is a bit confusing, and I'm sorry about that.  Basically, the `VolumeContainer` code here is using "vertex" data as opposed to "cell" data.  `vc_pos_index` is used for cell data, and we're using `vc_index` for vertex data.  `vc_pos_index` is only used in our contouring code, which is separate.